changelogs/unreleased/549-improve-docstrings.yml

```
@@ -0,0 +1,4 @@
+description: Fix typos in docstrings.
+issue-nr: 549
+change-type: patch
+destination-branches: [master, iso6]
```

I don't really see how these typos are linked to the referenced issue, but the issue-repo was missing.
```suggestion
description: Fix typos in docstrings.
issue-nr: 549
issue-repo: inmanta-license
change-type: patch
destination-branches: [master, iso6]
```

---

Ah, good point actually. Not linked at all apart from the fact that I fixed them while working on the referenced issue. I think it makes more sense to just remove the `issue-nr`.